app/channel.go

```diff
@@ -1912,8 +1912,15 @@ func (a *App) GetAllChannelsCount(c request.CTX, opts model.ChannelSearchOpts) (
 	return count, nil
 }
 
-func (a *App) GetDeletedChannels(c request.CTX, teamID string, offset int, limit int, userID string) (model.ChannelList, *model.AppError) {
-	list, err := a.Srv().Store.Channel().GetDeleted(teamID, offset, limit, userID)
+func (a *App) GetDeletedChannels(c request.CTX, teamID string, offset int, limit int, userID string, isAdmin bool) (model.ChannelList, *model.AppError) {
```

Comment:
You'll need to adapt the tests for this to pass.

Reply:
Hey @noxer, I have updated the tests accordingly.

Reply:
Perfect, thanks.